src/styles/globalStyles.js

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from "styled-components";
+
+export const GlobalStyle = createGlobalStyle`
```

Comment:
are we going to add any normalise or reset? or shall we leave it as it is to not mess with the styles?